pkg/didcomm/common/service/did_comm_msg.go

```diff
@@ -112,6 +122,31 @@ func (m DIDCommMsgMap) ID() string {
 	return res
 }
 
+// Encode converts struct to message
+func (m DIDCommMsgMap) Encode(v interface{}) error {
```

Comment:
Why does this function need to be exported?

Reply:
Done!